Argo/Types/JSON.swift

```diff
@@ -28,6 +28,15 @@ public extension JSON {
   }
 }
 
+extension JSON {
+  func jsonForKey(key: Swift.String) -> JSON {
```

Comment:
we initially had this as a subscript `json[key]` but decided that felt weird so made this function instead ... I'd also be ok with `objectForKey` that's pretty standard for Cocoa APIs